### DISCUSSION TITLE
AspirePress Mission, Vision and Roadmap 2025

### DISCUSSION THREAD

**Discussion by sarah-savage** [aspirepress/.github#20] [Ideas]
# AspirePress Mission, Vision and Roadmap 2025

## The Need For Clarity

2024 saw tremendous upheaval in the WordPress community, with forks and repos and new projects springing up all over the place. AspirePress is one of those projects. With 2025 approaching, it's more important than ever that we clarify our mission, vision and roadmap for the year.

This document seeks to outline what AspirePress will stand for, and what we will achieve going forward, in a clear way that everyone can understand, whether or not they fully agree.

## Existing Documentation

AspirePress has a [Vision and Values](https://aspirepress.org/philosophy/) page that outlines our vision:

> AspirePress exists to develop free resources for the WordPress community that diversifies, decentralizes and democratizes WordPress.

This is a great starting point to define the mission, vision and roadmap for AspirePress. We also list five values that we think are important:

* Collaboration
* Connection
* Communication
* Consensus
* Circulation (or distribution of our work)

## The Next Generation Mission and Vision of AspirePress

As WordPress evolves, so does AspirePress. It's important that we outline a mission and vision that highlights our commitment to the work.

### The Mission

The Mission of AspirePress is to develop free tools for the WordPress community that helps diversify, decentralize and democratize the ecosystem.

### The Vision For 2025

The Vision of AspirePress for 2025 is:

1. A free and democratic package repository that interfaces with WordPress' current APIs, serving as a drop-in replacement.
2. Implementation of an AspirePress-led and community-developed API specification for replacing the .org repository.
3. Creation of packages that enhance, enable and grow WordPress into a more modern and future-focused CMS.
4. Development of an effective governance structure that permits AspirePress to survive difficult times as well as good ones.
5. The development of partnerships with key stakeholders, including plugin and theme developers, hosting providers, implementation experts in WordPress, among others.

### Audience for AspirePress

AspirePress seeks to serve three core audiences as part of its work:

1. Developers or end users who are technical enough to want an alternative repository.
2. Hosting providers, who wish to offer AspirePress as an alternative to the status quo.
3. Plugin developers, including commercial plugin developers, who wish to distribute their work to others.

Our measure of success will be implementation of our product by the world. To that end, we will track very basic metrics about usage, and hope to show that usage is increasing over time, as we develop the components more fully.

## Roadmap For 2025

To execute on our goals, we need a clear roadmap for the future. This roadmap is by no means set in stone, and can change based on the ways in which the WordPress world changes, but reflects our commitment to executing on our mission and vision for 2025. Note that this roadmap only goes through June; in May it is the responsibility of the board to develop another half year roadmap.

### January

* Establish server architecture so that we can deploy the MVP of AspireCloud into production.
* Establish the AspirePress Foundation.

### February

* Conduct a limited beta test to 100 - 500 users for AspirePress.
* Conclude development on AspireSync.
* Develop the specification for a system of publishing plugins other than SVN or pulling them from .org
* Conduct outreach to 100 plugin developers about first-party publishing, and their needs.

### March

* Conduct outreach to another 100 plugin developers about first-party publishing
* Begin implementing the specification developed in February around publishing first-party plugins.
* General availability of AspireCloud to the public.
* Establish governance and the initial board of directors for AspirePress Foundation.

### April

* Finish implementation of the publication API and begin testing.
* Based on initial user feedback, develop a roadmap for AspireUpdate and AspireCloud 2.0.
* Board meets for the first time, sets governance policies.
* Outreach continues to another 100 plugin developers.

### May
* Development begins on AspireCloud 2.0 and AspireUpdate 2.0.
* Roadmap is developed for AspirePress, the flagship product which augments (but DOES NOT FORK) WordPress.
* Launch the publication API along with eligible first-party developers.
* Conduct an audit of plugins currently used and in-demand from users.

### June
* Establish fundraising goals and start fundraising for AspirePress
* Begin host outreach with an established user base
* Begin development on AspirePress product.
* Start revision work on publication API.

### Second Half of 2025

* Ship AspirePress packages
* Develop specification and roadmap for AspireCloud/AspireUpdate 3.0
* Onboard new plugin developers (goal: 500 by the end of the year)
* Consider creation of a marketplace of plugins and themes, or services around both.

**Comment by asirota** [comment]
Can I suggest assembling a marketing outreach team to get the necessary audience to beta test AP?

This work will require quite a bit of influencing and convincing. Basically winning others over.
